server_environment/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "server configuration environment files",
-    "version": "9.0.1.2.0",
+    "version": "10.0.1.2.0",
```

Comment:
> Semantic versioning dictates that you reset the numbers on the right when a number on the left of it has changed. This jives with that IMO

Of course. But precisely in an Odoo context, I find it different. For me, the version of the module is something independent of the version of Odoo (even if the two version number is merged).
But if this is good practice at OCA. No problem, I will change.